public/app/plugins/panel/histogram/module.tsx

```diff
@@ -54,7 +63,9 @@ export const plugin = new PanelPlugin<Options, FieldConfig>(HistogramPanel)
     standardOptions: {
       [FieldConfigProperty.Color]: {
         settings: {
-          byValueSupport: true,
+          byValueSupport: false,
+          bySeriesSupport: true,
+          preferThresholdsMode: false,
```

Comment:
this is a drive-by fix, which makes sure the default color mode is classic palette, not thresholds.